### PR TITLE
Add campaign scheduling endpoint

### PR DESCRIPTION
**Pull request by jacobherrington**
This PR adds support for the [Schedule API-Triggered Campaigns](https://www.braze.com/docs/api/endpoints/messaging/schedule_messages/post_schedule_triggered_campaigns/) endpoint!

### CODE REVIEW

File: spec/integrations/campaigns_spec.rb [hunk 1]
```diff
@@ -16,4 +16,16 @@
       expect(response.success?).to eq(true)
     end
   end
+
+  describe "trigger_schedule" do
+    it "sends an email", vcr: true do
+      response = api.trigger_campaign_schedule(
+        campaign_id: "a2d4ea65-5429-a527-9d20-f2d725e44bd9",
+        broadcast: true,
+        schedule: { time: Time.now.iso8601 }
```
Comment (jonallured) [inline spec/integrations/campaigns_spec.rb:25]: Oh nice one!! 😍

**Comment by jonallured** [comment]
Ok I think this is red because standardrb is complaining - maybe try this:

```
$ bundle exec standardrb --fix
```

**Comment by jacobherrington** [comment]
I'm removing the spec from this PR until we can decide how we want to proceed regarding VCR.

**Review by jonallured — APPROVED** [review]
LGTM!! 🎉

**Pull request merged**